setup.py

```diff
@@ -37,24 +39,62 @@ def run(self):
             print(message)
 
 
-include_dir = get_config_var("INCLUDEDIR")
-include_dirs = [include_dir] if include_dir is not None else []
-library_dir = get_config_var("LIBDIR")
-library_dirs = [library_dir] if library_dir is not None else []
```

Comment:
Lines 40-43 have been replaced by `get_include_dirs` and `get_library_dirs`.

I found this necessary when testing building a wheel in a `quay.io/pypa/manylinux2014_x86_64` docker container, as the `INCLUDEDIR` and `LIBDIR` were not correctly configured for the `yum` installed `udunits2-devel` package.

We can now set `UDUNITS2_INCDIR` and `UDUNITS2_LIBDIR` appropriately to override with custom locations